projects/storefrontlib/src/shared/components/star-rating/star-rating.component.ts

```diff
@@ -36,17 +37,18 @@ export class StarRatingComponent implements OnInit {
 
   iconTypes = ICON_TYPE;
 
-  constructor(private el: ElementRef) {}
+  constructor(private el: ElementRef, private renderer: Renderer2) {}
```

Comment:
Please consider making it backward compatible with `constructor(private el: ElementRef, private renderer?: Renderer2)`